cosmosdb/common/ps-account-firewall-create.ps1

```diff
@@ -0,0 +1,34 @@
+# Create an Azure Cosmos Account for SQL (Core) API with IP Firewall
```

Comment:
Not yet linked from doc page.

Reply:
Yes, cannot link until this is published.